src/content/contributing/_index.en.md

```diff
@@ -5,17 +5,23 @@ weight = 150
 pre = "<b>7. </b>"
 +++
 
-### Getting in touch with the community
 
-* Join the [developer mailing list](https://bit.ly/submariner-dev).
-* Join our [weekly meetings](https://docs.google.com/document/d/1qnZ2LpF_rXGfnYYPNTldQ4WbeEUxwnuQD-xTC6GbZdg/edit) and feel free to propose topics for discussion.
+### Talk to Us!
 
-### Pick up an issue to work on
+We would love to hear from you, how you are using Submariner, and what we can do to make it better. 
 
-### Or propose an enhancement via github issues
+#### [GitHub Project](https://github.com/submariner-io)
+Check out the project and consider contributing. Pick up an issue to work on or propose an enhancement by reporting a new issue; once your code is ready to be reviewed, you can propose a pull request. You can find a good guide about the GitHub workflow [here](https://git-scm.com/book/en/v2/GitHub-Contributing-to-a-Project).
 
-### Proposing a pull request
-Once your code is ready to be reviewed, you can propose a pull request.
+#### [#submariner](https://kubernetes.slack.com/archives/C010RJV694M)
+Share your ideas in the #submariner channel in Kubernetes' Slack. If you need it, you can [request an invite to K8S slack instance](https://slack.k8s.io/).
+
+#### [Community Calendar](https://calendar.google.com/calendar/r?cid=NHFuZGVoOGY0bzZ1ajlvZnBsczh1NWNlZ2tAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ)
+As a member of the Submariner Community, join any of our community meetings - no registration required. The weekly [Submariner Community Meeting](https://tinyurl.com/wfbx37q) (Tuesdays at 5:00pm CET) is a good place to start.
+
+
+#### [Mailing List](https://groups.google.com/forum/#!forum/submariner-dev)
+
+Join the developer mailing list.
 
-You can find a good guide about the github workflow here: [contributing to a github project](https://git-scm.com/book/en/v2/GitHub-Contributing-to-a-Project)
```

Comment:
I'd keep this bit somewhere, I believe everybody is not aware of the github workflow, at least I wasn't

Reply:
Oh it's there sorry :-)